troubleshoot/observability/explore-data.md

```diff
@@ -1,6 +1,10 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/observability/current/exploratory-data-visualizations.html
+applies_to:
+  stack: all
+  serverless:
+    observability: all
```

Comment:
I don't think _Explore data_ is available in serverless. @elastic/obs-docs can anyone confirm? 

```suggestion
```

Reply:
Confirmed